ArdusatSDK.cpp

```diff
@@ -906,80 +889,32 @@ boolean Orientation::read(Acceleration & acc, Magnetic & mag) {
 
   // Roll is rotation around x-axis (-180 <= roll <= 180)
   // Positive roll is clockwise rotation wrt positive x axis
-  roll = (float) atan2(acc.y, acc.z);
+  roll = (float) atan2(this->accel->y, this->accel->z);
 
   // Pitch is rotation around y-axis (-180 <= pitch <= 180)
   // Positive pitch is clockwise rotation wrt positive y axis
-  if (acc.y * sin(roll) + acc.z * cos(roll) == 0) {
-    pitch = acc.x > 0 ? (PI_F / 2) : (-PI_F / 2);
+  if (this->accel->y * sin(roll) + this->accel->z * cos(roll) == 0) {
+    pitch = this->accel->x > 0 ? (PI_F / 2) : (-PI_F / 2);
   } else {
-    pitch = (float)atan(-acc.x / (acc.y * sin(roll) + acc.z * cos(roll)));
+    pitch = (float)atan(-this->accel->x / (this->accel->y * sin(roll) + this->accel->z * cos(roll)));
   }
 
   // Heading is rotation around z-axis
   // Positive heading is clockwise rotation wrt positive z axis
-  heading = (float)atan2(mag.z * sin(roll) - mag.y * cos(roll),
-			         mag.x * cos(pitch) + mag.y * sin(pitch) * sin(roll) +
-				 mag.z * sin(pitch) * cos(roll));
+  heading = (float)atan2(this->mag->z * sin(roll) - this->mag->y * cos(roll),
+			                   this->mag->x * cos(pitch) + this->mag->y * sin(pitch) * sin(roll) +
```

Comment:
grr. it looks good in my editor. i think there were a few `\t` at the beginning of the line.